CMakeLists.txt

```diff
@@ -15,6 +15,12 @@ include(GNUInstallDirs)
 include(CMakeDependentOption)
 include(FeatureSummary)
 
+if(APPLE)
+  set(QRCODEGEN_OPTION_DEFAULT OFF)
```

Comment:
sigh, i'm wondering whether i oughtn't just remove libqrcodegen. nobody uses it, its purpose confuses people, and i could probably implement ISO/IEC 18004:2015 in an hour or so.

Reply:
https://github.com/dankamongmen/notcurses/issues/1282